Formula/minimesos.rb

```diff
@@ -0,0 +1,18 @@
+class Minimesos < Formula
+  desc "Testing infrastructure for Mesos frameworks"
+  homepage "https://minimesos.org/"
+  url "https://github.com/ContainerSolutions/minimesos/archive/0.10.2.tar.gz"
+  sha256 "16b198078c9384426f8849b08e92e173816c4d6626331bc6569aa5de89dffbdd"
+
+  unless system "docker ps -aq > /dev/null"
+    depends_on "docker"
```

Comment:
Please just use `depends_on "docker"`. If you wish to use your own Docker please consider adding a `:docker`/`DockerRequirement` to Homebrew/brew.